provider/macros/src/tests.rs

```diff
@@ -58,7 +51,7 @@ fn test_data_marker() {
 fn test_keyed_data_marker() {
     // #[data_struct(BarV1Marker = "demo/bar@1")]
     check(
-        vec![quote!(BarV1Marker = "demo/bar@1")],
+        quote!(BarV1Marker = "demo/bar@1"),
```

Comment:
Praise: It's good that we can check the whole token sequence instead of half-parsing it in the test.